store/sqlstore/emoji_store.go

```diff
@@ -66,50 +70,51 @@ func (es SqlEmojiStore) GetByName(ctx context.Context, name string, allowFromCac
 }
 
 func (es SqlEmojiStore) GetMultipleByName(names []string) ([]*model.Emoji, error) {
-	keys, params := MapStringsToQueryParams(names, "Emoji")
+	// Creating (?, ?, ?) len(names) number of times.
```

Comment:
squirrel handles this cases very well, is there any reason to not us squirrel here?

Reply:
I thought this was a simple enough case not to involve squirrel. It's just repeating `?`.